generate/convert.go

```diff
@@ -516,8 +516,23 @@ func (g *generator) convertDefinition(
 			Description: def.Description,
 			Values:      make([]goEnumValue, len(def.EnumValues)),
 		}
+		goNames := map[string]*goEnumValue{}
```

Comment:
I'm sure it doesn't matter, but maybe `make(map[string]*goEnumValue, len(def.EnumValues))`?